esp8266-fastled-webserver/common.h

```diff
@@ -0,0 +1,336 @@
+/*
```

Comment:
SIMPLE FILE REVIEW

This file will have multiple comments....

This provides a single, common header file that each source file may include.  It ensures any functions, data structures, etc. that were previously available remain available, even if the implementations now exist in separate `.cpp` files.